bank_account_project.py

```diff
@@ -1,169 +0,0 @@
-'''
-***All credit for the data sourced for this project goes to exchangerates.org.uk***
-
-This program simulates a transaction with an ATM machine. If the user decides to 
-withdraw money from their bank account, they are given the option to enter the 
-withdrawal amount in a foreign currency. The program will grab the most updated 
-currency exchange figures and return the amount remaining in the bank account 
-in GBP. 
-'''
-
-#This function scrapes conversion rates between GBP(£) and some of the most 
-#common currencies in the world. The function takes one argument, gbp, that 
-#must be either an integar or a float. 
-def currency_func(gbp):
-
-    import requests
-    from bs4 import BeautifulSoup
-    
-    URL = 'https://www.exchangerates.org.uk/'
```

Comment:
I like how you have capitalized this - so the reason this should have been capitalized, even if it wasn't a abbreviation - is because this is a constant. In Python, constants are by convention in Capital Snake Case. Example:

```
HOURS_IN_A_DAY = 24
```

However, since it is a constant, it doesn't need to be defined in the function every time, so this can be moved outside of the function.